static/app/components/profiling/aggregateFlamegraphPanel.tsx

```diff
@@ -7,14 +9,37 @@ import {t} from 'sentry/locale';
 import {FlamegraphStateProvider} from 'sentry/utils/profiling/flamegraph/flamegraphStateProvider/flamegraphContextProvider';
 import {FlamegraphThemeProvider} from 'sentry/utils/profiling/flamegraph/flamegraphThemeProvider';
 import {useAggregateFlamegraphQuery} from 'sentry/utils/profiling/hooks/useAggregateFlamegraphQuery';
+import usePageFilters from 'sentry/utils/usePageFilters';
 import {ProfileGroupProvider} from 'sentry/views/profiling/profileGroupProvider';
 
 export function AggregateFlamegraphPanel({transaction}: {transaction: string}) {
   const {data, isLoading} = useAggregateFlamegraphQuery({transaction});
+  const {
+    selection: {projects},
+  } = usePageFilters();
 
   const isEmpty = data?.shared.frames.length === 0;
+
+  const profileGroupInput = useMemo(() => {
+    if (!data) {
+      return null;
+    }
+
+    // TODO: this is a hack and should be coming from `vroom`
+    // without this our contextMenu links don't work
+    const profileGroupWithprofileId: Profiling.Schema = {
+      ...data,
+      metadata: {
+        ...data.metadata,
+        // without this our contextMenu links don't work
+        projectID: projects[0],
+      },
+    };
```

Comment:
see: https://github.com/getsentry/vroom/pull/212

Reply:
@eliashussary I think with the sentry sampled format we actually get access to the projectID from the response, I'd need to double check, but I think I saw it in there

Reply:
Ah right, different endpoint 🤦🏼